Makefile.builder

```diff
@@ -1,3 +1 @@
-ifeq ($(PACKAGE_SET),dom0)
```

Comment:
Please consider all supported rpm-based distributions. Specifically I think this will fail to on CentOS. If that's true, it's fine to exclude centos build for now. But it is not ok to have enabled package that doesn't build.